temporalio/worker/workflow_sandbox/_restrictions.py

```diff
@@ -99,6 +99,17 @@ class methods (including __init__, etc). The check compares the against the
     fully qualified path to the item.
     """
 
+    passthrough_all_modules: bool = False
+    """
+    Pass through all modules, do not sandbox any modules. This is the equivalent
+    of setting :py:attr:`passthrough_modules` as every module ever imported into
+    the workflow. This is unsafe. This means modules are never reloaded per
+    workflow run which means workflow authors have to be careful that they don't
+    import modules that do non-deterministic things. Note, just because a module
+    is passed through from outside the sandbox doesn't mean runtime restrictions
+    on invalid calls are not still applied.
```

Comment:
I don't understand any of this docstring; the same will be true of many potential users.